Generative AI with sub-concepts (#2043)
Codecademy/docs: content/ai/concepts/generative-ai/concepts/text/text.md

---
Title: 'Text'
Description: 'Generative AI can create blogs, write ad copy, create new content based on text input, and is great at summarizing and changing the style and tone of text content.'
Subjects:
  - 'Machine Learning'
  - 'Data Science'
Tags:
  - 'AI'
CatalogContent:
  - 'paths/machine-learning-engineer'
  - 'intro-to-chatgpt'
---

> **Generative text** is a common medium people a hear about. Just some of the applications include:

Author: Fixed.

---
Title: 'Text'

> Description: Generative AI can create blogs, write ad copy, create new content based on text input, and is great at summarizing and changing the style and tone of text content.

Reviewer: ```suggestion
Description: 'Generative AI can create blogs, write ad copy, create new content based on text input, and is great at summarizing and changing the style and tone of text content.'
```

Subjects:
  - 'Machine Learning'
  - 'Data Science'

>   - 'Large Language Model'

Reviewer: ```suggestion
```

>   - 'Transformers'

Reviewer: ```suggestion
```

> Tags:

Reviewer: Only AI is listed on the [Tags list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).
All the other terms should be added to that file in this PR.
